File: URI/1 - INICIANTE/Python/1165 - NumeroPrimo.py

z = int(input())
while z > 0:
    num = int(input())
    div = 0
    for y in range(1, num+1):
        if(num % y == 0):
            div += 1
    if div == 2:
        print('{} eh primo'.format(num))
    else:
        print('{} nao eh primo'.format(num))
    z -= 1

File: URI/1 - INICIANTE/C/1165 - NumeroPrimo.c
#include <stdio.h>

int main()
{
	int x, y, num, div = 0;
	
	scanf(" %d", &x);
	
	while(x > 0){
		scanf(" %d", &num);
		div = 0;
		for(y = 1; y <= num; y++){
			if(num % y == 0){
				div+=1;
			}
		}
		if(div == 2){
			printf("%d eh primo\n", num);
		}else{
			printf("%d nao eh primo\n", num);
		}
		x--;
	}
	
	return 0;
}
